internal/project/random.go

```diff
@@ -0,0 +1,53 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package project
+
+import (
+	"crypto/rand"
+	"encoding/base64"
+	"encoding/hex"
+	"fmt"
+)
+
+// RandomHexString generates a random string of the provided length.
+func RandomHexString(len int) (string, error) {
```

Comment:
Is this taken from Verification? I think we have a unit test that goes along with it (and I suppose we can have verification depend on this in the future)

Reply:
It is. I can do that as a follow-up. I want to do a full diff and see if there are other things that should be moved over